.circleci/config.yml

```diff
@@ -77,12 +77,6 @@ commands:
           root: /tmp
           paths:
             - workspace
-  setup_gcloud_storage:
-    # TODO(danielrozenberg): remove this after deploy-bot can use CircleCI artifacts.
-    steps:
-      - run:
-          name: 'Setup Google Cloud Storage'
-          command: ./.circleci/setup_storage.sh
```

Comment:
I assume you also meant to delete this file?

Reply:
Correct. Done.